pycqed/analysis_v2/base_analysis.py

```diff
@@ -199,9 +204,39 @@ def process_data(self):
     def prepare_plots(self):
         pass
 
-    def run_fitting(self):
+    def prepare_fitting(self):
         pass
 
+    def run_fitting(self):
+        # This function does the fitting and saving of the parameters
```

Comment:
If you start and end comments with triple quotes (''') they will show up as docstrings when you do ? on the method or when (if we ever do this) auto generate docs in the future. 

👍  for the comments